src/plugins/intel_gpu/src/kernel_selector/kernels/prior_box/prior_box_kernel_ref.cpp

```diff
@@ -82,11 +82,15 @@ bool PriorBoxKernelRef::Validate(const Params& params, const optional_params& op
         return false;
     }
 
-    const auto& priorBox8arams = dynamic_cast<const prior_box_params&>(params);
-    if (priorBox8arams.inputs.size() != 2) {
+    const auto& priorBoxParams = dynamic_cast<const prior_box_params&>(params);
+    if (priorBoxParams.inputs.size() != 2) {
         return false;
     }
 
+    // Current ref kernel doesn't support clustered version
+    if (priorBoxParams.is_clustered) {
+        return false;
```

Comment:
If we dont support a spec, then could you add an exception handling at the creation of the op?

Reply:
I added the assertion in CreateOp(). Thanks!